esphome/components/m5stack4relay/__init__.py

```diff
@@ -0,0 +1,32 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import CONF_ID
+
+CODEOWNERS = ["@KoenBreeman"]
+DEPENDENCIES = ["i2c"]
+
+MULTI_CONF = True
+
+CONF_I2C_ADDR = 0x26
+
+CONF_M5Stack4Relay_ID = "m5stack4relay_id"
```

Comment:
Should be all uppercase
```suggestion
CONF_M5STACK4RELAY_ID = "m5stack4relay_id"
```